www/src/pages/components/checkbox/react/index.mdx

```diff
@@ -15,97 +15,57 @@ Checkboxes can be checked, unchecked, or in an indeterminate state.
 The `isChecked` prop determines if a checkbox is checked.
 
 ```jsx
-class CheckboxExample extends React.Component {
-    constructor(props) {
-        super(props);
-
-        this.state = {
-            isChecked: undefined,
-        };
-
-        this.onChange = this.onChange.bind(this);
-    }
-
-    onChange(isChecked) {
-        this.setState({
-            isChecked,
-        });
-    }
-
-    render() {
-        const { isChecked } = this.state;
-
-        return (
-            <Checkbox isChecked={isChecked} onChange={this.onChange}>
-                Send me promotional emails
-            </Checkbox>
-        );
-    }
+function CheckboxExample() {
```

Comment:
I think it'd be good to also include a single example (possibly right after this one?) that shows how to do it with classes. We're still using classes heavily, so I can see it being confusing to folks if we _only_ document with hooks.

(The code examples do look a million times better with hooks though!)

Reply:
done